e2e-tests/batch-mode-tests.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
Nice thing about this little incantation from my perspective is that it’s more generically applicable; my work machine is a Mac, so `/bin/bash` is actually a quite old version of bash for me whereas this allows it to run in a newer version of bash installed via homebrew. I suppose this only needs to work on Travis, though.

Reply:
I see, okay, I thought something was not working on Travis environment which forced us to make this change. Either way is fine with me.

Reply:
Well, one of the `sh` files didn't have any shebang at all, so it appeared to be running on bash in `sh` mode, but it could've just as easily been fixed with `#!/bin/bash`.